system_tests/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM ubuntu:20.04
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# prep env so that required packages are cached
+RUN apt-get -y update && apt upgrade -y && \
+    apt-get -y install sudo build-essential curl wget jq nfs-common nfs-kernel-server udev
+
+# get packages that will be installed during bootstrapping
+RUN apt-get -y install vim ksh tcsh zsh libssl-dev ncurses-dev libpam-dev net-tools \
+               libhwloc-dev dkms tcl-dev automake autoconf libtool librrd-dev       \
+               libapr1-dev libconfuse-dev apache2 libboost-dev libdb-dev tcsh       \
+               libncurses5-dev libpam0g-dev libxt-dev libmotif-dev libxmu-dev       \
+               libxft-dev libhwloc-dev man-db lvm2 python r-base libblas-dev        \
+               libffi-dev libxml2-dev mdadm libgcrypt20-dev libmysqlclient-dev      \
+               libevent-dev iproute2 python3 python3-pip libatlas-base-dev          \
+               libglvnd-dev linux-headers-aws iptables libcurl4-openssl-dev         \
+               python3-parted
```

Comment:
are you installing these packages explicitly here rather than having them install from the chef run in order to cache this layer? Trying to understand the value in keeping this duplicate list of packages

Reply:
Yes -- for running repeatedly locally having these cached saves significant time.